prog/vm/host_nexus.rb

```diff
@@ -40,6 +40,19 @@ def before_run
   end
 
   label def wait_bootstrap_rhizome
+    reap
+    hop_download_boot_images if leaf?
+    donate
+  end
+
+  label def download_boot_images
+    Option::BootImages.each do |image|
+      bud Prog::DownloadBootImage, {"image_name" => image.name, "custom_url" => nil, "should_learn_storage" => false}, :download
```

Comment:
If you update the `custom_url` helper as follows, there's no need to specify `custom_url` here.
```ruby
  def custom_url
     @custom_url ||= frame.fetch("custom_url", nil)
  end
```